charts/kubescape-cloud-operator/values.yaml

```diff
@@ -516,3 +516,43 @@ otelCollector:
     limits:
        cpu: 1
        memory: 1Gi
+
+# Values for the Kubescape Storage service that Kubescape uses for its internal
+# purposes like storage
+kubescapeStorage:
+  enabled: true
+
+  # Values or the Aggregated APIServer
+  k8sApiserver:
+    name: "storage-aggregated-apiserver"
+
+    deployment:
+      labels:
+        app.kubernetes.io/name: "aggregated-apiserver"
+        app.kubernetes.io/component: "apiserver"
+        app.kubernetes.io/part-of: "kubescape-storage"
+
+      replicaCount: 1
+      image:
+        pullPolicy: "IfNotPresent"
+        repository: "quay.io/matthiasb_1/storage"
```

Comment:
This image should be published to the Kubescape org.
Also, it should support `arm64` and `amd64`.

Reply:
will work on it with @vladklokun

Reply:
Yeah, that’s going to happen only when we create proper CI for Storage.